SimTracker/Common/src/SiG4UniversalFluctuation.cc

```diff
@@ -222,7 +207,7 @@ double SiG4UniversalFluctuation::SampleFluctuations(const double momentum,
   //
   if (suma > sumalim)
   {
-    p1 = 0., p2 = 0 ;
+    double p1, p2;
```

Comment:
I see if (something) p1=blah then if (something2) p2=blah, then bah=p1*a+p2*b. So you definitely need to initialize p1 and p2.